eng/common/TestResources/New-TestResources.ps1

```diff
@@ -200,19 +200,29 @@ try {
         # Make sure the user is logged in to create a service principal.
         $context = Get-AzContext;
         if (!$context) {
-            Log "You are not logged in; connecting to 'Azure SDK Developer Playground'"
+            $subscriptionName = $SubscriptionId
+            $wellKnownSubscriptions = @{
+                'faa080af-c1d8-40ad-9cce-e1a450ca5b57' = 'Azure SDK Developer Playground'
+                'a18897a6-7e44-457d-9260-f2854c0aca42' = 'Azure SDK Engineering System'
```

Comment:
Do we have another one that we use for live testing?

Reply:
You're right. I was originally (incorrectly) thinking that was the "Engineering System" one, but I see we have:

```text
Azure SDK Test Resources       2cd617ea-1866-46b1-90e3-fffb087ebf9b
```

I'll add it. The Java pipeline failed for unrelated reasons anyway and has to be restarted.